src/sentry/api/endpoints/sentry_app_requests.py

```diff
@@ -21,6 +21,19 @@ def format_request(self, request, sentry_app):
             "responseCode": request.get("response_code"),
         }
 
+        if "error_id" in request and "project_id" in request:
+            try:
+                project = Project.objects.get_from_cache(id=request["project_id"])
+                # Make sure the project actually belongs to the org that owns the Sentry App
+                if project.organization_id == sentry_app.owner_id:
+                    error_url = "/organizations/{}/projects/{}/events/{}".format(
```

Comment:
Should this have a trailing slash? Most of our URLs have them.

Reply:
@markstory it doesn't seem to matter either way but I'll add for consistency